appengine/twilio/README.md

```diff
@@ -1,9 +1,5 @@
 # Java Twilio Voice and SMS Sample for Google App Engine Standard Environment
 
-**Java 7 runtime support on App Engine standard was [deprecated](https://cloud.google.com/appengine/docs/deprecations/java7) on
```

Comment:
This warning seems like it might be best to leave in the README until the sample it removed?